homeassistant/components/growatt_server/sensor.py

```diff
@@ -983,7 +983,7 @@ def update(self):
                 self.data = inverter_info
```

Comment:
As the construct of the inverter is the same as the tlx, I doubt that the inverter logic works without ["data"]

but this is out of scope of this PR